019_check_if_vowel_or_consonent.js

```diff
@@ -0,0 +1,18 @@
+// Write a program to check if a character is a vowel or consonant
+
+const checkvowelOrConsonent = (alphabet) => {
+    for (let i = 0; i < 5; i++) {
+        if (alphabet === vowelArray[i]) {
+            vowel = true;
+            break;
+        } else {
+            continue;
+        }
+    }
+    return vowel;
+}
+
+let vowelArray = ["a", "e", "i", "o", "u"];
+let vowel = false;
```

Comment:
why you need this varialbe

Reply:
I have updated the revised code. Please review.